entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eu # Increase bash strictness
+
+if [[ -n "${GITHUB_WORKSPACE}" ]]; then
+  cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit
+fi
+
+export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
+
+export REVIEWDOG_VERSION=v0.11.0-nightly20210104+207b245
```

Comment:
Can you use v0.11.0 instead of nightly? Or do we have any reason to use the nightly version?

Reply:
I need to use nightly here till v0.12.0 is released. This is because this action depends on https://github.com/reviewdog/errorformat/commit/3c79a1ba48222113d5b55a238813ca255f4812e3. We can change it to a specific version after the release.